pyoptsparse/pyOpt_optimization.py

```diff
@@ -1084,13 +1083,13 @@ def processObjtoVec(self, funcs: Dict1DType, scaled: bool = True) -> NumpyType:
             if objKey in funcs:
                 try:
                     f = np.squeeze(funcs[objKey]).item()
-                except ValueError:
-                    raise Error(f"The objective return value, '{objKey}' must be a scalar!")
```

Comment:
What is the nuance of this syntax? I got it before when the two errors were different, is it just for a more explicit handling of the error?

Reply:
Yeah, the error from numpy is probably less helpful than this specific message. In a later date I would like to clean up shape handling once and for all, see e.g. #150  and #151.